src/common/Input.tsx

```diff
@@ -137,6 +156,30 @@ const Input: React.FC<InputProps> = (props) => {
           color: #d2d2d2;
           text-decoration: line-through;
         }
+        .password-input {
+          padding-right: calc(var(--font-size-regular) * 2.8);
+        }
+        .input-wrapper {
+          position: relative;
+        }
+        .visibility-toggle {
+          position: absolute;
+          right: 0px;
+          top: 0px;
+          line-height: 17.5px;
+          font-size: var(--font-size-regular);
+          padding: 13px;
```

Comment:
Add `cursor: pointer;` to `visibility-toggle` so that we get the "hand pointing up" cursor icon when hovering on it.